lib/elasticsearch/indexing/bulk.ex

```diff
@@ -100,6 +100,7 @@ defmodule Elasticsearch.Index.Bulk do
     bulk_page_size = index_config[:bulk_page_size] || 5000
     bulk_wait_interval = index_config[:bulk_wait_interval] || 0
     action = index_config[:bulk_action] || "create"
+    bulk_path = index_config[:bulk_path] || "/_bulk"
```

Comment:
@ssbb This is technically a breaking change for anyone who is currently using an old version of Elasticsearch. I would prefer to make  the default `"/_doc/_bulk"` here until I'm ready to release v2.0.0. 

```suggestion
    bulk_path = index_config[:bulk_path] || "/_doc/_bulk"
```

Reply:
@danielberkompas Done! I just misread README about compatibility with older versions.